recipes/pygnome/meta.yaml

```
@@ -0,0 +1,103 @@
+{% set name = "PyGnome" %}
+{% set version = "1.1.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NOAA-ORR-ERD/PyGnome/archive/v{{ version }}.tar.gz
+  sha256: ad14d089932098bcd0b063f4c08fb6a0836de4747edec0497397671f1830e03b
+
+build:
+  skip: True  # [py<39 or py>311 or python_impl == 'pypy']
+  missing_dso_whitelist:
+    - "*/lib_gnome.dll"         # [win]
+    - "*/point_in_polygon.dll"  # [win]
+  number: 0
+  script: cd py_gnome && {{ PYTHON }} -m pip install ./ -vv
+
+requirements:
+  build:
+    - python                              # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - numpy
+    - cython
+    - netCDF4
```

These should also be in `host`. Maybe you need them here for the `build_platform != target_platform` but really only for that.
```suggestion
    - numpy
    - cython
    - netCDF4
```

---

ok, I made the change.  I put these three items in *both* the build and host sections, and nothing seems to have been broken.

---

In build, they should have a `# [build_platform != target_platform]` comment.